### PR TITLE
Docs + other views

### PR DESCRIPTION
**Pull request by anastasia**
- fixed capapi views
- rewrote/updated docs
- added whitelisted jurisdictions filter
- added django-pipeline settings

screen shot 2018-04-02 at 5 43 49 pm

### CODE REVIEW

File: capstone/fabfile.py [hunk 1]
```diff
@@ -400,7 +400,7 @@ def show_slow_queries():
     try:
         rows = cursor.fetchall()
         output = "*capstone slow query report*\n"
-    except:
+    except Exception:
         print(json.dumps({'text': 'Could not get slow queries'}))
```
Comment (anastasia) [inline capstone/fabfile.py:404]: updated flake8 is angry at naked except clauses

Comment (bensteinberg) [reply capstone/fabfile.py:404]: Wait, why did flake8 complain to you and not to me?

Comment (anastasia) [reply capstone/fabfile.py:404]: I accidentally updated flake8. It'll complain in the future.

**Comment by jcushman** [comment]
Can't wait!

**Pull request merged**